source/main/MainThread.cpp

```diff
@@ -94,7 +94,8 @@ MainThread::MainThread():
 	m_exit_loop_requested(false),
 	m_base_resource_loaded(false),
 	m_application_state(Application::STATE_NONE),
-	m_next_application_state(Application::STATE_NONE)
+	m_next_application_state(Application::STATE_NONE),
+    m_is_mumble_created(false)

```

Comment:
Happens to the best of us, too: https://github.com/NVIDIAGameWorks/GraphicsSamples/blob/master/samples/vk10-kepler/DedicatedAllocationVk/DedicatedAllocationVk.h#L61-L71
:laughing: